k2/csrc/array_ops_test.cu

```diff
@@ -1342,6 +1337,144 @@ TEST(OpsTest, InvertMonotonicDecreasingTest) {
   }
 }
 
+template <typename T>
+void ArrayPlusTest() {
+  ContextPtr cpu = GetCpuContext();  // will be used to copy data
+  for (auto &context : {GetCpuContext()}) {
```

Comment:
Shall we also enable cuda test?

Reply:
Oh, sorry, forget to add it back after testing.

Reply:
Fixed, thanks!